group-aws/integration.md

```diff
@@ -11,24 +24,22 @@ exist are in test/integration/targets
 * Resource modification
 * Resource deletion
 
-When adding new features, those features should have tests.
-
 # Github PRs
 There are some PRs with tests:
 
 | Module               | PR                                                     |
 |----------------------|--------------------------------------------------------|
-| lambda               | [24951](https://github.com/ansible/ansible/pull/24951) |
 | rds                  | [25646](https://github.com/ansible/ansible/pull/25646) |
 | rds_param_group      | [25345](https://github.com/ansible/ansible/pull/25345) |
 
 # State of the codebase
 
 Existing modules with tests:
 * `aws_api_gateway`
+* `aws_lambda`
 * `aws_s3`
 * `ec2_ami`
-* `ec2_elb_lb`
```

Comment:
Since the naming was undone and now ec2_elb_lb and elb_class_lb both have tests, this line should be put back.

Reply:
Thanks @s-hertel, this is fixed now